vendor/vendor.json

```diff
@@ -1659,6 +1665,12 @@
 			"path": "golang.org/x/net/lex/httplex",
 			"revision": "dd2d9a67c97da0afa00d5726e28086007a0acce5",
 			"revisionTime": "2017-02-23T02:43:42Z"
+		},
+		{
+			"checksumSHA1": "538I8ghUdvZa25NxgMUDpq14Qic=",
+			"path": "gopkg.in/check.v1",
```

Comment:
Do we need to include this? It looks like juju/ratelimit only uses this for its tests, which we shouldn't be running, right?

Reply:
I used govendor to add juju/ratelimit. It added this automatically. If I remove it, next time when somebody runs govendor again, may it be added back? I will check it.